docs/release-1.2/existing-rhdh.md

```diff
@@ -64,121 +91,156 @@ metadata:
 EOF
 ```
 The value of `.data.npmrc` points to https://npm.registry.redhat.com.
-For testing RC plugin versions, update to `cmVnaXN0cnk9aHR0cHM6Ly9ucG0uc3RhZ2UucmVnaXN0cnkucmVkaGF0LmNvbQo=` (points to https://npm.stage.registry.redhat.com and can be accessed internally). If there is a need to point to multiple registries, modify the content of the secret's data from:
-```
+For testing RC plugin versions, update to `cmVnaXN0cnk9aHR0cHM6Ly9ucG0uc3RhZ2UucmVnaXN0cnkucmVkaGF0LmNvbQo=` (which points to https://npm.stage.registry.redhat.com and can be accessed internally within Red Hat). If there is a need to point to multiple registries, modify the content of the secret's data from:
+```yaml
   stringData:
     .npmrc: |
       registry=https://npm.registry.redhat.com
 ```
 to
-```
+```yaml
   stringData:
     .npmrc: |
       @redhat:registry=https://npm.registry.redhat.com
       @<other-scope>:registry=<other-registry>
 ```
 
 ### dynamic-plugins ConfigMap
-This ConfigMap houses the configuration for enabling and configuring dynamic plugins. To incorporate the orchestrator plugins, append the following configuration to the **dynamic-plugins** ConfigMap:
+This ConfigMap houses the configuration for enabling and configuring dynamic plugins in RHDH. 
+
+> Note: Be sure to review [this section](#identify-latest-supported-plugin-versions) to determine the latest supported Orchestrator plugin `package:` and `integrity:` values, and update the dynamic-plugin ConfigMap entries accordingly. The samples in this document may not be the latest. 
+
+To incorporate the Orchestrator plugins, append the following configuration to the **dynamic-plugins** ConfigMap:
 
 ```yaml
-  - disabled: false
-    package: "@redhat/backstage-plugin-orchestrator-backend-dynamic@1.2.0"
-    integrity: sha512-lyw7IHuXsakTa5Pok8S2GK0imqrmXe3z+TcL7eB2sJYFqQPkCP5la1vqteL9/1EaI5eI6nKZ60WVRkPEldKBTg==
-    pluginConfig:
-      orchestrator:
-        dataIndexService:
-          url: http://sonataflow-platform-data-index-service.sonataflow-infra
-  - disabled: false
-    package: "@redhat/backstage-plugin-orchestrator@1.2.0"
-    integrity: sha512-FhM13wVXjjF39syowc4RnMC/gKm4TRlmh8lBrMwPXAw1VzgIADI8H6WVEs837poVX/tYSqj2WhehwzFqU6PuhA==
-    pluginConfig:
-      dynamicPlugins:
-        frontend:
-          janus-idp.backstage-plugin-orchestrator:
-            appIcons:
-              - importName: OrchestratorIcon
-                module: OrchestratorPlugin
-                name: orchestratorIcon
-            dynamicRoutes:
-              - importName: OrchestratorPage
-                menuItem:
-                  icon: orchestratorIcon
-                  text: Orchestrator
-                module: OrchestratorPlugin
-                path: /orchestrator
+      - disabled: false
```

Comment:
Was it rendered badly? I mean, is this change intentional?

Reply:
Perhaps to allow it being copied and pasted easily without the need to take care of indentation? 
The output of `View file` for this section looks good though.

Reply:
I did purposefully adjust for the expected indentation, so maybe that is it.